config/database.yml.github_actions

```diff
@@ -0,0 +1,25 @@
+# @note This file is only for use in GitHub Actions.  If you need to make a
```

Comment:
Existing travis configuration:
https://github.com/rapid7/metasploit-framework/blob/8aaae387ea67c908c71472b3a239078fb3675ac5/config/database.yml.travis

Reply:
Travis seems to connect over a mounted socket, whilst the Github docker image requires a password and specifying the host as localhost